blog/2021/02/julia-1.6-highlights.md

```diff
@@ -0,0 +1,335 @@
+
+If this doesn’t work for some reason, Julia 1.6 also introduces the
+`NetworkOptions.jl` stdlib: this package acts as a central place for network
+configuration options that can be controlled by various environment variables
+and which are used to modify the behavior of networking libraries like `libcurl`
+and `libgit2` in a consistent way. For example, if you want to turn off HTTPS
+host verification entirely, you can do `export JULIA_SSL_NO_VERIFY_HOSTS="**"`
+in your shell and both the Downloads and LibGit2 packages will not perform host
+verification when downloading over HTTPS. There are various other options
+available in NetworkOptions, including:
+
+- `JULIA_SSL_CA_ROOTS_PATH` to provide a custom PEM file of CA roots
+- `SSH_KNOWN_HOSTS_FILES` to use non-standard locations for SSH known hosts
+- `JULIA_*_VERIFY_HOSTS` variables for fine-grained control over which hosts
+  should or shouldn’t be verified over various transports, including TLS and SSH
+
+These options are now consistently respected across all network-facing code that
+ships with Julia itself and we will be working with package developers to
+encourage them to use `NetworkOptions` for configuration of libraries such as
+mbedtls and others. This will allow consistent configuration of networking
+options across the entire Julia ecosystem.
+
+## CI Robustness
+
+_Jeff Bezanson & Keno Fischer_
+
+This release cycle we spent quite a bit of time paying down technical debt in
+the form of intermittent test failures in our continuous integration (CI)
+process. Like all responsible software projects these days, we run our full
+build and test suite for every commit and for every proposed change. If the
+tests fail, you stop the presses until the problem is fixed — either by
+reverting a change, committing a new fix, or revising a proposed patch until it
+passes. Given this simple policy, it’s difficult to see how a project could be
+ambushed by persistent test failures. And yet that’s exactly what happened to
+us: over time, we ended up in a state where a high percentage of test runs
+failed, usually with just a single obscure test case failing.
+
+Several factors contributed to this predicament. First, the base Julia test
+suite is quite large and covers a wide range of functionality, from parsing and
+compiling to linear algebra, package management, sockets, handling file system
+events, and more. With that much surface area, we were likely to end up with a
+handful of rare bugs, or failures due to overly-fragile tests. We run easily
+over a hundred builds per day, so even failure rates of 0.1% would appear often
+enough to cause problems. Timing-sensitive tests are a classic example, e.g.
+testing that a one-second timeout indeed happens after approximately one second.
+On hosted VMs in particular, timing can be far more variable than what you would
+ever see on dedicated hardware. A one-second timeout can, unfortunately, take
+more than 60 seconds on a heavily loaded VM.
```

Comment:
This could use a concluding sentence or two to summarize the progress. This does a good job of laying out the problem, and mentions that you put welcome work into it, but it doesn't say anything about how many cases were fixed, or how much of a change there was in the frequency of such problems, etc.

Reply:
Working on it! I put this aside for a couple weeks but I'll add more soon.

Reply:
Your addition is great.